tests/core/test_organization_plan.py

```
@@ -0,0 +1,630 @@
+    assert row["reason"] == "Categorized into Docs"
+
+
+def test_error_message_summarizes_first_three_conflicts() -> None:
+    from file_organizer.core.plan import (
+        PlanConflict,
+        PlanConflictType,
+        PlanValidationResult,
+    )
+
+    conflicts = [
+        PlanConflict(PlanConflictType.SOURCE_MISSING, f"/tmp/{i}", "gone")  # noqa: test-hardcoded-paths
```

Ruff (0.15.21)

[error] Probable insecure usage of temporary file or directory: "/tmp/"

(S108)